notification-service/src/main/resources/bootstrap.yml

```diff
@@ -0,0 +1,5 @@
+spring:
+  cloud:
+    config:
+      uri: http://localhost:8888/config
```

Comment:
Values that change depending on the environment (in this case: host, port) should be moved to be in the .env file.

Reply:
Done.